data/category-anticensorship

```diff
@@ -27,13 +28,16 @@ getoutline.org
 gfw.report
 guardianproject.info
 hiddify.com
+ipleak.com
+ipleak.net
```

Comment:
Lines 21, 31, and 32 are not appropriate for inclusion in the anti-censorship category because they merely serve as websites to test your network environment, not to circumvent Internet censorship.

Reply:
I removed Lines 21, 31, and 32 ("dnsleaktest.com", "ipleak.com" and "ipleak.net"). However, in that regard, line 16 ("browserleaks.com") is the same, and I believe it should also be removed. I think a category for these kinds of websites should be created.